src/TrackArt.cpp

```diff
@@ -332,30 +372,40 @@ bool TrackArt::DrawClipTitle(
 
 bool TrackArt::DrawAudioClipTitle(
```

Comment:
I think this function should be moved into `WaveTrackAffordanceControls.cpp` which is the sole user of it.